gaia-checkbox.js

```diff
@@ -235,7 +252,8 @@ function getLinkedCheckbox(label) {
   if (!label) { return; }
   var id = label.getAttribute('for');
   var el = id && document.getElementById(id);
-  return el && el.tagName === 'GAIA-CHECKBOX' ? el : null;// || label.querySelector('gaia-checkbox');
+  // || label.querySelector('gaia-checkbox');
```

Comment:
This line was originally added to provide support for chekboxes nested inside `<label>` being toggle-able. I can't remember why it wa commented out. We should re-enable it, or remove it.